packages/synapse-react-client/src/components/ChallengeTeamWizard/ChallengeTeamWizard.tsx

```diff
@@ -230,23 +219,11 @@ const ChallengeTeamWizard: React.FunctionComponent<
     }
   }, [accessToken, userProfile, projectId, challenge])
 
-  /**
-   * When creating a new team, wait for the team to be registered to the challenge
-   * and for invited members to be invited before proceeding to confirmation view.
-   */
-  useEffect(() => {
-    if (inviteMembersSuccess && registerChallengeSuccess) {
-      setConfirming(false)
-      handleStepChange(step.confirmStep as StepsEnum)
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [registerChallengeSuccess, inviteMembersSuccess, step])
-
```

Comment:
Remove this effect that took us to the next step when all of the requests completed. We will directly move to the step in an appropriate callback